spec/cypress/app_commands/scenarios/schools/multiple_schools.rb

```diff
@@ -0,0 +1,7 @@
+cohort = FactoryBot.create(:cohort, :current)
+first_school = FactoryBot.create(:school, name: "Test School 1", slug: "111111-test-school-1")
+FactoryBot.create(:school_cohort, school: first_school, cohort: cohort)
+second_school = FactoryBot.create(:school, name: "Test School 2", slug: "111112-test-school-2")
+FactoryBot.create(:school_cohort, school: second_school, cohort: cohort)
```

Comment:
Maybe it would be worth adding a third school that isn't added to the user to check that it _isn't_ displayed?